packages/ui-components/src/components/dropdown/dropdown.tsx

```diff
@@ -51,11 +52,44 @@ export class KvDropdown implements IDropdown, IDropdownEvents {
 		this.openStateChange.emit(this.isOpen);
 	};
 
+	private closePositionAutoUpdate;
+
+	componentDidRender() {
+		const inputContainer = this.el.shadowRoot.querySelector('#dropdown-input');
+		const input = inputContainer.shadowRoot.querySelector('input');
+		const dropdownList = this.el.shadowRoot.querySelector('#dropdown-list') as HTMLElement;
+
+		if (this.isOpen) {
+			this.closePositionAutoUpdate = autoUpdate(input, dropdownList, () => {
+				computePosition(input, dropdownList, {
+					placement: 'bottom',
+					middleware: [
+						offset(8),
+						flip({
+							padding: 15,
+							fallbackPlacements: ['top']
+						})
+					]
+				}).then(({ x, y }) => {
+					Object.assign(dropdownList.style, {
```

Comment:
Could we mutate the object directly here?
dropdownList.style.left = `${x}px`;

Or atleast we should prefer object spread over assign.

Reply:
I think so. I used this because it's the suggested in the FloatingUI lib.

Reply:
I would assume it is because of older browser support. It's safer than mutating directly since the object might not exist (?), but slower than the other approaches, it's up yo you if the change makes sense, just commenting so we don't start using `Object.assign` as a rule of thumb.